research-hub-web/src/app/app.component.ts

```diff
@@ -1,45 +1,67 @@
-
 import { filter, distinctUntilChanged } from 'rxjs/operators';
-import { Component, OnDestroy, OnInit, ViewEncapsulation, ViewChild, AfterViewInit, ElementRef, NgZone } from '@angular/core';
-import { CategoryId, OptionsService, OptionType } from './services/options.service';
+import { 
+  Component, 
+  OnDestroy, 
+  OnInit, 
+  ViewEncapsulation, 
+  ViewChild, 
+  AfterViewInit, 
+  ElementRef, 
+  NgZone 
+} from '@angular/core';
 import { SearchBarService } from './components/search-bar/search-bar.service';
 import { NavigationEnd, Router } from '@angular/router';
-import { Subscription, Observable, fromEvent } from 'rxjs';
-import { debounceTime, tap, pluck, map } from 'rxjs/operators';
+import { Subscription, fromEvent } from 'rxjs';
+import { debounceTime } from 'rxjs/operators';
 import { ResearchHubApiService } from './services/research-hub-api.service';
 import { AnalyticsService } from './services/analytics.service';
 import { isPlatformBrowser } from '@angular/common';
 import { ChangeDetectorRef } from '@angular/core';
 import { format } from 'date-fns';
 import { LoginService } from '@uoa/auth';
-
-import { HeaderService } from './components/header/header.service';
 import { Location } from '@angular/common';
 import { AppComponentService } from './app.component.service';
 import { Title } from '@angular/platform-browser';
 import { ScrollDispatcher } from '@angular/cdk/scrolling';
-import {
-  trigger,
-  state,
-  style,
-  animate,
-  transition
+import { 
+  trigger, 
+  state, 
+  style, 
+  animate, 
+  transition 
```

Comment:
Delete these extra spaces just to stop other dev's prettier/linter making unnecessary changes next time they save this file.

Reply:
- [ ] These still need deleting